### DISCUSSION TITLE
How to Install on "Web Space"

### DISCUSSION THREAD

**Discussion by AWFordJr** [LycheeOrg/Lychee#2719] [Q&A] [answered]
I have looked this up on YouTube but its for installing it on a home server & I pay for web space so I have no desire to host it from home. I have been using the older version 3.1.6 for many years just because its been the easiest to install. Simply upload the extracted files to my web space where I want it, give the database information, and it asks me for a user/pass. Now its constantly giving me the stupid "error message" about a missing api or whatever no matter how many times I upload it & try to install it again. I love this gallery, but its so annoying to setup sometimes. I have never had anything past 3.1.6 work for me because its not the same or I just dont get which directly I should be uploading. Plus I doubt I need to upload 13K files... I have a feeling there is a lot more files because that is for home servers right? Which folder gets uploaded?

**Comment by ildyria** [comment, marked as answer]
Sorry, I have not seen your message before, you need to download the Lychee.zip archive from the release.
Yes there are A LOT of files, and yes you need all of them otherwise it will not work.

Between version 3 and 6 of Lychee there has been a lot of changes, strong improvement in the robustness of the software, way more functionalities, see our releases notes: https://lycheeorg.github.io/docs/releases.html
